app/backend/controllers/login-signup.js

```diff
@@ -137,14 +181,20 @@ module.exports.resetPassword = async (request, response) => {
     response.status(400).send({errmsg: 'New password is required in the request body'})
   }
 
-  checkPasswordLength(request.body.password, response)
-
+  if(checkPasswordLength(request.body.password, response)){
+    response.status(400).send({ errmsg: 'Password length must be at least 6.' })
+    return
```

Comment:
To prevent the execution of the rest of the function would be ensured with an `else` block better.